### PR TITLE
Allow Custom Tabs UI to be customizable

### PR DESCRIPTION
**Pull request by lbalmaceda**
Displaying the page title should be the default and only behavior, along with showing the current page domain. ~This PR doesn't add the ability to turn this off (hide title).~
A builder class is provided to customize the CT UI options.

Related: https://github.com/auth0/Auth0.Android/issues/106

### CODE REVIEW

File: auth0/src/main/java/com/auth0/android/provider/CustomTabsController.java [hunk 1]
```diff
@@ -124,6 +124,7 @@ public void run() {
                 Log.d(TAG, "Launching URI. Custom Tabs available: " + available);
 
                 final Intent intent = new CustomTabsIntent.Builder(session.get())
+                        .setShowTitle(true)
```
Comment (hzalaz) [inline auth0/src/main/java/com/auth0/android/provider/CustomTabsController.java]: Should this be optional? I am not too keen to force all users for a single person gh issue

Comment (lbalmaceda) [inline auth0/src/main/java/com/auth0/android/provider/CustomTabsController.java]: Ok. I'll add in this same PR a builder/style to customize this and the toolbar color.

**Comment by twaddington** [comment]
I'd really like to see this merged. What's the current status of this change?

**Comment by lbalmaceda** [comment]
@twaddington it's missing the methods to hide/show the title and change the toolbar color. I'll try to add them before eow.

**Comment by twaddington** [comment]
@lbalmaceda thanks! I personally wouldn't mind if the title just always showed and was not configurable. Not sure how others would like that though.

**Comment by lbalmaceda** [comment]
@twaddington I know but we should let devs make that decision. 👍

**Comment by cocojoe** [comment]
Let's seem some screenshots.

**Comment by lbalmaceda** [comment]
@cocojoe coco http://g.recordit.co/i3pdJxee2l.gif

**Comment by twaddington** [comment]
Y'all are awesome. Thanks!

**Comment by lbalmaceda** [comment]
Please test it and give us feedback. Thanks

**Pull request merged**